fields/types/markdown/MarkdownType.js

```diff
@@ -93,7 +93,7 @@ markdown.prototype.addFilterToQuery = function (filter) {
  * Formats the field value
  */
 markdown.prototype.format = function (item) {
-	return item.get(this.paths.html);
+	return item.get(this.paths.md);
```

Comment:
@mxstbr His fix for #3001 got included (accidentally). 
@WelltokCaleb Better to drop that commit for this PR and leave it to #3002